lib/manager/bazel/extract.js

```diff
@@ -101,8 +101,10 @@ function extractPackageFile(content) {
       dep.depName = depName;
       dep.currentValue = currentValue || commit.substr(0, 7);
       dep.purl = 'pkg:go/' + importpath;
-      if (remote) {
+      if (remote && !remote.match(/\.git/)) {
```

Comment:
The check must be more strict. It should only match URLS like https://github.com/foo/bar and nothing else